provision/vvv-init.sh

```diff
@@ -54,6 +55,31 @@ fi
 cp -f "${VVV_PATH_TO_SITE}/provision/vvv-nginx.conf.tmpl" "${VVV_PATH_TO_SITE}/provision/vvv-nginx.conf"
 sed -i "s#{{DOMAINS_HERE}}#${DOMAINS}#" "${VVV_PATH_TO_SITE}/provision/vvv-nginx.conf"
 
+
+# Proxy images from live site
+if [[ "${LIVE_URL}" != http* ]]; then
+	LIVE_URL=''
+else
+	LIVE_URL=$(cat <<'END_HEREDOC'
+
+
+	# Directives to send expires headers and turn off 404 error logging.
+	location ~* .(js|css|png|jpg|jpeg|gif|ico)$ {
+		expires 24h;
+		log_not_found off;
+		try_files \$uri \$uri\/ @production;
+	}
+	
+	location @production {
+		resolver 8.8.8.8;
```

Comment:
So this will break VVV for some users, in particular, those in China and other oppressive regimes

Reply:
but we need that part?